elytron/src/main/java/org/wildfly/extension/elytron/ModifiableKeyStoreDecorator.java

```diff
@@ -85,8 +85,20 @@ public void registerOperations(ManagementResourceRegistration resourceRegistrati
 
     static class ReadAliasesHandler extends ElytronRuntimeOnlyHandler {
 
+        static final SimpleAttributeDefinition RECURSIVE = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.RECURSIVE, ModelType.BOOLEAN, true)
+                .setAllowExpression(false)
```

Comment:
Just minor, but setAllowExpression can be set to `true` ?

Reply:
@baranowb Ignore this comment, @fjuma explained to me that since these are only used in operations there is no need to allow expressions for them